cookiecutter-frontend-app/cookiecutter.json

```diff
@@ -0,0 +1,8 @@
+{
+  "repo_owner": "edx",
+  "repo_name": "frontend-app-template",
+  "author_name": "{{cookiecutter.repo_owner}}",
+  "description": "What is this project?",
+  "version": "0.1.0",
+  "port": "8080"
```

Comment:
We actually have an example of the ports falling out of sync [1], so doing it through the template makes sure we catch them all.

- [1] https://github.com/edx/frontend-app-learning/commit/93ccdf829b6be7c7c8316c912403d5b14db2bc92